meta_model_building/utility.py

```diff
@@ -0,0 +1,71 @@
+""" This module contains methods called in the main script for obtaining data:
+get_opt_params.py"""
+
+import copy
+from tpot.config.regressor import regressor_config_dict
+from tpot.config.regressor_light import regressor_config_dict_light
+
+# model lists and dicts
+implemented_model_list = [
+    'regression', 'regression_light', 'knn',
```

Comment:
A nice time for working with git: if you break out one item per line, you can very easily see when a diff is adding or removing items. Also, if you end with a comma (allowed in Python) you can add to the list (or remove) and only the added/removed lines show on the diff.

So something like:

```python
implemented_model_list = [
    'regression', 
    'regression_light', 
    'knn',
    'decision_tree', 
    'linear', 
    'SVR', # <-- Notice ending comma
]
```